docker-compose.yaml

```diff
@@ -325,7 +325,7 @@ services:
       - SYS_PTRACE
   
   console:
-    image: docker.io/vesoft/nebula-console:3.5.0
+    image: docker.io/vesoft/nebula-console:v3.5.0
```

Comment:
could you please put `vesoft/nebula-console:v3.5` instead? Thanks!

Reply:
no problem😜